### DISCUSSION TITLE
Multiple forms & Next.JS

### DISCUSSION THREAD

**Discussion by mattletw** [quillforms/quillforms#125] [Q&A]
Hi all,

I have a questions around implementation in combination with Next.JS.
I've succeeded to somehow implement Quillforms in an existing website. My plan is to have multiple forms. However after the first form is succeeded and the user is redirected to the second form, the form is empty.

Is this expected behavior? Where should the registration of the core blocks in Next.JS happening?

**Comment by mdmag** [comment]
Hello @mattletw,

I will try to reproduce this. 
We might have an issue with using multiple forms because all use the same store, so might will try resetting the store when the second form appears.

**Reply by mattletw** [reply]
@mdmag Thanks for you answer.

How can I reset the store? Is there a method for that?

**Comment by mattletw** [comment]
@mdmag 

How can I reset the store?
